### ISSUE TITLE
Use `logging` and only warn on incomplete packet read, not incomplete block read

### ISSUE THREAD

**Issue by eackermann** [kemerelab/jagular#1]
I don't actually think that

```
print("Read {} complete packets but requested {} packets".format(num_samples_read, block_size))
```
is an interesting occurrence. This is almost always not seen by the user, since JFM gets the remaining packets from the next file. So this is only mildly interesting in the last file, but even then, not really.

What IS interesting to log and/or catch, is if we ever read an incomplete packet, not an incomplete block.

At any rate, we should avoid littering our code with print statements, and should use warnings at worst, or have an optional verbose mode, but logging is highly preferred.

**Comment by jchutrue** [comment]
Agreed--will change that when we have a logging solution figured out.
